include/crm/stonith-ng.h

```diff
@@ -521,6 +521,17 @@ stonith_api_time_helper(uint32_t nodeid, bool in_progress)
     return (*st_time_fn) (nodeid, NULL, in_progress);
 }
 
+/**
+ * Does the given agent describe a stonith resource that can exist?
+ *
+ * \param[in] agent     What is the name of the agent?
+ * \param[in] timeout   Timeout to use when querying.  If 0 is given,
+ *                      use a default of 120.
+ *
+ * \return A boolean
+ */
+gboolean stonith_agent_exists(const char *agent, int timeout);
```

Comment:
Code that directly includes this header such as dlm_stonith is getting:
```
/usr/include/pacemaker/crm/stonith-ng.h:535:1: error: unknown type name 'gboolean'; did you mean 'bool'?
  gboolean stonith_agent_exists(const char *agent, int timeout);
  ^~~~~~~~
  bool
```
We should probably either use bool or includes glib.h in this header.

Reply:
Doh! At least it's just in master. I need to add some tests like this to the CI or at least the release testing.

The public fencing API avoids glib currently because one of the original goals was to allow stonithd to be used outside the context of pacemaker. But that never went anywhere, so I don't know if it's still a useful requirement.

This particular function signature was chosen to be comparable to resources_agent_exists(), but since the two types work identically and there's no need for glib otherwise, I think bool makes sense here.

Reply:
Easy enough to fix, one minute.

Reply:
See #1713